packages/control-property-editor/test/unit/panels/outline/OutlinePanel.test.tsx

```diff
@@ -288,6 +284,7 @@ describe('OutlinePanel', () => {
             scale: 1,
             outline: model,
             filterQuery: filterInitOptions,
+            scenario: 'ADAPTATION_PROJECT',
```

Comment:
same as above

Reply:
Changed in: https://github.com/SAP/open-ux-tools/pull/1422/commits/83134d16a0fff1526e4dddf61c067912f7a5bdf2